File: node-server/softwareApplication.js
const {CreativeWork} = require("./creativeWork.js");

class SoftwareApplication extends CreativeWork{
	update(idSoftwareApplication, alternativeHeadline, commentCount, copyrightYear, inLanguage, isAccessibleForFree, applicationCategory, applicationSubCategory, applicationSuite, fileSize){
		super.update(idSoftwareApplication, alternativeHeadline, commentCount, copyrightYear, inLanguage, isAccessibleForFree);
        this.applicationCategory = applicationCategory;
        this.applicationSubCategory = applicationSubCategory;
		this.applicationSuite = applicationSuite;
		this.fileSize = fileSize;
	}
}

module.exports = {SoftwareApplication};

File: python-server/application/model/softwareApplication.py
from application.model.creativeWork import CreativeWork

__author__ = 'Lucía Blanco Rodríguez'


class SoftwareApplication(CreativeWork):
    def __init__(self, id, alternativeHeadline, commentCount, copyrightYear, inLanguage, isAccessibleForFree, applicationCategory, applicationSubCategory, applicationSuite, fileSize):
        super().__init__(id, alternativeHeadline, commentCount, copyrightYear, inLanguage, isAccessibleForFree)
        self.applicationCategory = applicationCategory
        self.applicationSubCategory = applicationSubCategory
        self.applicationSuite = applicationSuite
        self.fileSize = fileSize
